pkg/test/integration/common/helpers/cluster.go

```diff
@@ -163,3 +122,20 @@ func (c *Cluster) getSecret(ref *v1.SecretReference, MachineClassName string) (*
 	}
 	return secretRef, err
 }
+
+func (c *Cluster) VerifyControlClusterNamespace(isControlSeed string, controlClusterNamespace string) error {
```

Comment:
Add a comment describing what the method does. It is an exposed method, so this is required to be done.

Reply:
Do you have suggestions for the comment?